libraries/tools/kotlin-stdlib-gen/src/templates/Elements.kt

```diff
@@ -607,7 +607,12 @@ object Elements : TemplateGroupBase() {
     } builder {
         inline(Inline.Only)
         doc { "Returns the first ${f.element} matching the given [predicate], or `null` if no such ${f.element} was found." }
-        sample("samples.collections.Collections.Elements.find")
+        specialFor(CharSequences) {
+            sample("samples.text.Strings.find")
+        }
+        specialFor(ArraysOfUnsigned) {
```

Comment:
The original `sample` call should be left as is, without any additional `specialFor` predicates, and the call generating a sample specialized for char sequences should be placed right after it (wrapped into a predicate):
```
sample("samples.collections.Collections.Elements.find")
specialFor(CharSequences) {
    sample("samples.text.Strings.find")
}
```